apps/condo/domains/ticket/access/Ticket.js

```diff
@@ -111,6 +115,30 @@ async function canManageTickets (args) {
             return ticket.client === user.id
         }
     } else {
+        if (isBulkRequest && operation === 'create') return false
+
+        if (isBulkRequest && operation === 'update') {
+            if (itemIds.length !== uniq(itemIds).length) return false
+            if (itemIds.length !== originalInput.length) return false
+
+            if (!originalInput.every((updateItem) => {
+                for (const key in updateItem.data) {
+                    if (!BULK_UPDATE_ALLOWED_FIELDS.includes(key)) return false
+                }
+                return true
+            })) return false
+
```

Comment:
We may have crooked bulk updates of employees. This is true. I want to roll out this feature and then take on the task of updating accesses. However, if this is really bad, then I can make a crutch. We will check that only one user is updated. And only he will be in the check for compliance with the employee's organization - the ticket organization

Reply:
Let me know what is the best way to proceed